client/ayon_nuke/api/colorspace.py

```diff
@@ -0,0 +1,338 @@
+        displays = view_profile["display"].split(COLOR_VALUE_SEPARATOR)
+    else:
+        displays = []
+
+    # generate all possible combination of display/view
+    display_views = []
+    for view in views:
+
+        if displays:
+            for display in displays:
+                display_views.append(
+                    create_viewer_profile_string(
+                        view.strip(), display.strip(), path_like=False
+                    )
+                )
+        else:
+            display_views.append(view.strip())
+
+    for dv_item in display_views:
+        # format any template tokens used in the string
+        dv_item_resolved = StringTemplate(dv_item).format_strict(
+            formatting_data)
+        log.debug("Resolved display and view: `{}`".format(dv_item_resolved))
+
+        # making sure formatted colorspace exists in running session
+        if dv_item_resolved in get_display_and_view_colorspaces(root_node):
+            return dv_item_resolved
+
+
+def get_formatted_display_and_view_as_dict(
+        view_profile, formatting_data, root_node=None):
+    """Format display and view profile into dict.
+
+    This method is formatting a display and view profile. It is iterating
+    over all possible combinations of display and view colorspaces. Those
+    could be separated by COLOR_VALUE_SEPARATOR defined in constants.
+
+    If Anatomy template tokens are used but formatting data is not provided,
+    it will try any other available variants in next position of the separator.
+
+    This method also validate that the formatted display and view profile is
+    available in currently run nuke session ocio config.
+
+    Example:
+        >>> from ayon_nuke.api.colorspace import get_formatted_display_and_view_as_dict  # noqa
+        >>> view_profile = {
+        ...     "view": "{context};sRGB",
+        ...     "display": "{project_code};ACES"
+        ... }
+        >>> formatting_data = {
+        ...    "context": "01sh010",
+        ...    "project_code": "proj01"
+        ...}
+        >>> display_and_view = get_formatted_display_and_view_as_dict(
+        ...    view_profile, formatting_data)
+        >>> print(display_and_view)
+        {"view": "01sh010", "display": "proj01"}
+
+
+    Args:
+        view_profile (dict): view and display profile
+        formatting_data (dict): formatting data
+        root_node (Optional[nuke.Node]): root node
+
+    Returns:
+        dict: formatted display and view profile in dict
+            ex: {"view": "sRGB", "display": "ACES"}
+    """
+    if not root_node:
+        root_node = nuke.root()
+
+    views = view_profile["view"].split(COLOR_VALUE_SEPARATOR)
+    displays = view_profile["display"].split(COLOR_VALUE_SEPARATOR)
+
+    # display could be optional in case nuke_default ocio config is used
+    if view_profile["display"]:
+        displays = view_profile["display"].split(COLOR_VALUE_SEPARATOR)
+    else:
+        displays = []
```

Comment:
Double split of `"display"`.
```suggestion

    # display could be optional in case nuke_default ocio config is used
    displays = []
    if view_profile["display"]:
        displays = view_profile["display"].split(COLOR_VALUE_SEPARATOR)
```